chart/etcd-cloud-operator/templates/statefulset.yaml

```diff
@@ -0,0 +1,127 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "etcd-cloud-operator.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "etcd-cloud-operator.name" . }}
+    helm.sh/chart: {{ include "etcd-cloud-operator.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  serviceName: {{ include "etcd-cloud-operator.fullname"  . }}
+  replicas: {{ default 3 .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "etcd-cloud-operator.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "etcd-cloud-operator.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          volumeMounts:
+          {{- if .Values.persistence.enabled }}
+          - name: data
+            mountPath: {{ .Values.config.etcd.dataDir }}
+          {{- end }}
+          {{- if eq .Values.config.snapshot.provider "file" }}
+          - name: snapshot
+            mountPath: /var/lib/snapshots
+          {{- end}}
+          - name: config
+            mountPath: /etc/eco
+            readOnly: true
+          env:
+          - name: INITIAL_CLUSTER_SIZE
+            value: {{ default 3 .Values.replicaCount | quote }}
```

Comment:
I don't think you're using that value in the code anymore

Reply:
Yup. Removed it. Thanks!